### DISCUSSION TITLE
fwupd 2.0.0 release date/ WWCB certification /update successfully reports

### DISCUSSION THREAD

**Discussion by PatxHo** [fwupd/fwupd#7107] [General]
Dear Richard,

Currently, we are working on LVFS of 3 Acer Docks, 2 of them are T34 (thunderbolt) and U33.
 I found that "fwupdtool get-devices --plugins **rts54hub** " in fwupd version **2.0.0** can show up its 2 hubs, 
but not showing up in other versions of **1.9.xx**.
V2.0.0
![image](https://github.com/fwupd/fwupd/assets/113101054/3f1f795e-1ab1-48bf-9179-a38063c711be)
V1.9.x
![image](https://github.com/fwupd/fwupd/assets/113101054/66e4ecb7-2164-4bc9-bfbc-e5d183595f80)

Q1. Do you have any info about when fwupd 2.0.0 will be released?

Q2. Google engineers will only pick cabs which were already been pushed in the **Stable Stage** and merge them into their ChromeOS image build, right? 

Q3. How long does Google take to sync up with the same fwupd version (e.g. v2.0.0) after LVFS releases a new version?  

Q4. Is it correct that only using the ubuntu preload version of "fwupdmgr get-devices" can pop up the successful reports after updating successfully? Because I cannot execute fwupdmgr in the distribution version (e.g. 1.9.16), it prompts "Unsupported daemon versoin xxx, clinet verson is 1.9.16"

Thank you.
[1.9.5_get-devices_-vv.txt](https://github.com/fwupd/fwupd/files/14974283/1.9.5_get-devices_-vv.txt)
[2.0.0_get-devices_-vv.txt](https://github.com/fwupd/fwupd/files/14974285/2.0.0_get-devices_-vv.txt)

**Comment by hughsie** [comment]
> Q1. Do you have any info about when fwupd 2.0.0 will be released?

We've not got a release planned, although we do create 1.9.x releases about every 3 weeks. If you need a specific commit backported into 1_9_X (the branch we use for the 1.9.x releases) please just let us know.

I've done what I think you need in https://github.com/fwupd/fwupd/pull/7110 -- although in future this should probably be done by Ricky -- I guess it was just easily forgotten.

> Google engineers will only pick cabs which were already been pushed in the Stable Stage

The firmware needs to be in "stable" and also needs to have been tested on ChromeOS -- see https://lvfs.readthedocs.io/en/latest/chromeos.html

> How long does Google take to sync up with the same fwupd version (e.g. v2.0.0) after LVFS releases a new version?

I don't think it'll pick up 2.0.0 for quite some time, but it does pick up the stable releases (1.9.x) usually a couple of weeks after they are released upstream

> it prompts "Unsupported daemon versoin xxx, clinet verson is 1.9.16"

It sounds like you're running a mixture of the snap and debian packages perhaps? If you're running the snap version you need to remove the dpkg version. Alternately, use Fedora where the distro package is kept up to date.

Richard.

**Comment by PatxHo** [comment]
Thank you for the answers and huge help.
Much appreciated! Almighty Richard. 😄
